docs/details/clickhouse.md

```diff
@@ -0,0 +1,41 @@
+# ClickHouse
+
+
+You can use an external Clickhouse database instance outside the PMM Server container running on other hosts.
+
+## Environment variables
```

Comment:
please also add it here: https://github.com/percona/pmm-doc/blob/08a1633b8d86da102ad1dda81557e1489731232f/docs/setting-up/server/docker.md#environment-variables

Reply:
done